package.json

```diff
@@ -9,194 +9,124 @@
-    "eslint": "1.10.3",
-    "eslint-config-airbnb": "0.1.0",
-    "eslint-loader": "^1.0.0",
-    "eslint-plugin-import": "^0.8.0",
-    "eslint-plugin-react": "^3.5.0",
-    "extract-text-webpack-plugin": "^0.9.1",
-    "font-awesome": "^4.4.0",
+    "auth0": "^2.1.0",
+    "auth0-lock": "^9.0.1",
+    "babel-polyfill": "6.7.4",
+    "babel-runtime": "6.6.1",
+    "bcrypt": "0.8.5",
+    "body-parser": "1.15.0",
+    "bootstrap-sass": "^3.3.6",
+    "bootstrap-sass-loader": "^1.0.10",
+    "compression": "1.6.1",
+    "cors": "2.7.1",
+    "deep-equal": "1.0.1",
+    "es6-promisify": "4.0.0",
+    "express": "4.13.4",
+    "express-jwt": "3.3.0",
+    "font-awesome": "^4.5.0",
     "font-awesome-webpack": "0.0.4",
-    "json-loader": "^0.5.4",
-    "karma": "^0.13.10",
-    "karma-cli": "^0.1.1",
-    "karma-mocha": "^0.2.0",
-    "karma-mocha-reporter": "^1.1.1",
-    "karma-phantomjs-launcher": "^0.2.1",
-    "karma-sourcemap-loader": "^0.3.5",
-    "karma-webpack": "^1.7.0",
-    "mocha": "^2.3.3",
-    "node-sass": "^3.4.2",
-    "phantomjs": "^1.9.18",
-    "phantomjs-polyfill": "0.0.1",
-    "react-a11y": "^0.2.6",
-    "react-addons-test-utils": "^0.14.0",
-    "react-transform-catch-errors": "^1.0.0",
-    "react-transform-hmr": "^1.0.1",
-    "redbox-react": "^1.1.1",
-    "redux-devtools": "^3.0.0-beta-3",
-    "redux-devtools-dock-monitor": "^1.0.0-beta-3",
-    "redux-devtools-log-monitor": "^1.0.0-beta-3",
-    "sass-loader": "^3.1.2",
-    "sinon": "^1.17.2",
-    "strip-loader": "^0.1.0",
-    "style-loader": "^0.13.0",
-    "tachyons-sass": "^0.1.0",
-    "timekeeper": "0.0.5",
-    "webpack": "^1.12.9",
-    "webpack-dev-middleware": "^1.4.0",
-    "webpack-hot-middleware": "^2.5.0"
-  },
-  "engines": {
-    "node": "5.3.0",
-    "npm": "3.3.12"
+    "googleapis": "4.0.0",
+    "graphiql": "0.6.6",
+    "graphql": "0.4.18",
+    "immutable": "3.7.6",
+    "isomorphic-fetch": "2.2.1",
+    "joi": "8.0.5",
+    "jsonwebtoken": "5.7.0",
+    "jwt-decode": "^2.0.1",
+    "material-ui": "0.15.0-alpha.2",
+    "node-env-flag": "0.1.0",
+    "node-uuid": "1.4.7",
+    "piping": "0.3.0",
+    "react": "0.14.8",
+    "react-bootstrap": "^0.28.5",
+    "react-copy-to-clipboard": "^4.0.2",
+    "react-css-modules": "^3.7.6",
+    "react-dnd": "2.1.4",
+    "react-dnd-html5-backend": "2.1.2",
+    "react-dom": "0.14.8",
+    "react-dom-stream": "0.5.0",
+    "react-helmet": "^3.0.0",
+    "react-redux": "4.4.1",
+    "react-router": "2.0.1",
+    "react-router-redux": "4.0.1",
+    "react-tap-event-plugin": "0.2.2",
+    "redux": "3.3.1",
+    "redux-form": "4.2.2",
+    "redux-immutablejs": "0.0.8",
+    "redux-logger": "2.6.1",
+    "redux-optimistic-ui": "0.4.0",
+    "redux-promise": "0.5.3",
+    "redux-socket-cluster": "0.5.1",
+    "redux-thunk": "2.0.1",
+    "rethinkdbdash": "2.2.18",
+    "serve-favicon": "^2.3.0",
+    "socketcluster": "git://github.com/mattkrick/socketcluster.git#242b38a8660af47b0f9212206230e93d041ea5b6",
```

Comment:
@jordanh i think this is cleaner than making folks jump through hoops in the readme.